cloudmarker/events/azsqldatabasedisabledtdeevent.py

```diff
@@ -0,0 +1,95 @@
+"""Microsoft Azure SQL DB Transparent Data Encryption (TDE) event.
+
+This module defines the :class:`AzSQLDatabaseDisabledTDEEvent` class that
+identifies if a SQL Database has TDE disabled . This plugin works on the
+SQL DB properties found in the ``ext`` bucket of ``sql_db`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzSQLDatabaseDisabledTDEEvent:
```

Comment:
This plugin is a good candidate for having a built-in config key in `baseconfig.py`. Could you please add it so that it can be referred to easily in user's configuration?

Reply:
I have added `AzSQLDatabaseTDEEvent` to `baseconfig.py` as a built-in config key.